deepchecks/nlp/utils/weak_segments.py

```diff
@@ -39,7 +39,7 @@ def get_relevant_data_table(text_data: TextData, data_type: str, columns: Union[
 
     if n_top_features is not None and n_top_features < features.shape[1]:
```

Comment:
isnt this conflicting with the above change?

Reply:
This happens before my line